component/azstorage/utils.go

```diff
@@ -596,3 +596,21 @@ func removeLeadingSlashes(s string) string {
 	}
 	return s
 }
+
```

Comment:
Tag filter is not there right now, but later at some point we will add it hence keeping this code as commented for now.